spec/support/acceptance/servicenow/mock_instance.rb

```diff
@@ -20,13 +20,11 @@ class MockServiceNowInstance < Sinatra::Base
       SSLEnable: true,
       SSLCertName: [['CN', 'Sinatra']]
 
-  # Handles authorization
-  use Rack::Auth::Basic do |user, password|
-    user == 'mock_user' && password == 'mock_password'
-  end
-
   before do
     content_type 'application/json'
+    unless ['Basic bW9ja191c2VyOm1vY2tfcGFzc3dvcmQ=', 'Bearer mock_token'].include?(env['HTTP_AUTHORIZATION'])
+      halt 401, to_error_response('Authorization Failed')
```

Comment:
I'd make this error message a bit clearer. `'Authorization failed -- expected token to be X but got Y'`. Saves headaches down the line.